config/billing/pricing_data.yml

```diff
@@ -33,6 +33,7 @@ eu-west-1:
   rds_mysql_m4_4xlarge: "1.545"
   rds_mysql_m4_4xlarge_ha: "3.09"
   elasticache_t2_micro: "0.018"
+  elasticache_m5_large: "0.172"
```

Comment:
You'll also need to update the `config.json.erb` template to add a new plan that references these values.

Reply:
I've updated the config template now.